docs/glossary.md

```diff
@@ -57,7 +57,7 @@ To access resources such as SGTS services and GCC via this device, you must onbo
 
 </a>
 
-If you are using a non-SE GSIB device, every time you log in to your device, you will be prompted to enter your BitLocker PIN. You can access the internet and intranet using this device.
+If you are using a non-SE GSIB device, every time you log in to your device, you are prompted to enter your BitLocker PIN. You can access the internet and intranet using this device.
```

Comment:
The keyword for this definition should be "Non-SE GSIB" instead of "non-SE GSIB". Can i suggest the "SE GSIB" to swap order with "Non-SE GSIB" i.e. SE GSIB comes first then Non-SE GSIB? 

Fact check, SE GSIB no need BitLocker PIN? if not why is this highlighted here

Reply:
Glossary is arranged in alphabetical order hence the current order.
BitLocker PIN removed from the definition.